ParisBNF/abb/BNFabb156.xml

```diff
@@ -86,67 +86,70 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      
                      <msItem xml:id="ms_i1.5">
                         <locus from="12vb" to="15ra"/>
-                        <title type="complete" ref="LIT6824CommentariesOldTest#Numbers">Commentary on Numbers</title>
+                        <title type="complete" ref="LIT4117Interpretation#Numbers">Commentary on Numbers</title>
                         <textLang mainLang="gez"/>                       
                         <incipit xml:lang="gez"><hi rend="rubric"/></incipit>
                      
                   </msItem>
                      
                      <msItem xml:id="ms_i1.6">
                         <locus from="15ra" to="16rb"/>
-                        <title type="complete" ref="LIT6824CommentariesOldTest#Deuteronomy">Commentary on Deuteronomy</title>
+                        <title type="complete" ref="LIT4117Interpretation#Deuteronomy">Commentary on Deuteronomy</title>
                         <textLang mainLang="gez"/>                       
                         <incipit xml:lang="gez"><hi rend="rubric"/></incipit>
                      </msItem>
                   
                   <msItem xml:id="ms_i1.7">
                      <locus from="16rb" to="18ra"/>
-                     <title type="complete" ref="LIT6824CommentariesOldTest#Hosea">Commentary on Hosea</title>
+                     <title type="complete" ref="LIT4117Interpretation#Joshua">Commentary on Joshua</title>
```

Comment:
I do not see Joshua here https://github.com/BetaMasaheft/Works/pull/996/files#diff-f37ea814e88207b1e97a149d11af6a053b9ca02746f39ecbe7d87cf55ebedf7d

Reply:
fixed